scipy/optimize/tests/test_zeros.py

```diff
@@ -5,7 +5,7 @@
 from numpy.testing import (assert_warns, assert_, 
                            assert_allclose,
                            assert_equal)
-from numpy import finfo
+from numpy import finfo, exp as np_exp, sin as np_sin, asarray
```

Comment:
I would `import numpy as np` and then use `np.exp` etc. Probably should have been done for `finfo`, but that's for another day.